src/cmd/support_archive/resource_query.go

```diff
@@ -0,0 +1,100 @@
+package support_archive
+
+import (
+	"reflect"
+
+	"github.com/Dynatrace/dynatrace-operator/src/api/v1beta1"
+	"github.com/Dynatrace/dynatrace-operator/src/kubeobjects"
+	"github.com/Dynatrace/dynatrace-operator/src/webhook"
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+type resourceQueryGroup struct {
+	resources []schema.GroupVersionKind
+	filters   []client.ListOption
+}
+
+type resourceQuery struct {
+	groupVersionKind schema.GroupVersionKind
+	filters          []client.ListOption
+}
+
+func getQueries(namespace string) []resourceQuery {
```

Comment:
just to double check, we dont want to collect "everything" in the first iteration just to be safe ?

because there are still stuff that could be collected, like yamls of injected pods that would be really useful during debug

Reply:
offline answer => maaaybe in the future

Reply:
goal for now is to collect information about direct operator components. let's get that archive into a good state, and then check how we can extend it to include more information, like injected pods, in the future.